integration_tests/intrinsics_01.py

```diff
@@ -0,0 +1,15 @@
+from numpy import array
+
+def any_01() -> None:
+    x: bool[2]
+
+    x= array([False, False])
+    assert not any(x)
+
+    x = array([False, True])
+    assert any(x)
```

Comment:
I think Python's any accepts not only arrays but anything like lists, tuples. Also, it doesn't accepts arguments like `dim` and `mask`.

Reply:
I wanted this: https://github.com/lfortran/lfortran/pull/1485 changes in LPython, so I submitted a PR with these changes.

We will do one thing, we will skip `any` for now and implement it later.